lib/Basic/SourceLoc.cpp

```diff
@@ -695,7 +695,11 @@ unsigned SourceManager::getExternalSourceBufferID(StringRef Path) {
     return It->getSecond();
   }
   unsigned Id = 0u;
-  auto InputFileOrErr = swift::vfs::getFileOrSTDIN(*getFileSystem(), Path);
+  auto InputFileOrErr =
+  swift::vfs::getFileOrSTDIN(*getFileSystem(), Path,
+                             /* FileSize */ -1,
+                             /* RequiresNullTerminator */ true,
+                             /* isVolatile */ this->editorMode);
```

Comment:
clang-format? This feels like it is misindented.

Reply:
Good catch, fixed.